ui/src/components/settings/BasicSettings.vue

```diff
@@ -442,6 +442,8 @@
 
                         let newlang = Utils.getLang();
                         this.$moment.locale(newlang);
+                        const messages = await import(`../../translations/${newlang}.json`)
+                        this.$v_i18n.global.setLocaleMessage(newlang, messages[newlang]);
```

Comment:
Can you explain this a little?

If I understand correctly, this is to load the locale when saving.
Since the change of locale is stored in `localStorage`, the change is taken.

I believe this change will result in loading the translation file twice.